tips/TIP-0046/tip-0046.md

```diff
@@ -0,0 +1,997 @@
+        <table>
+          <tr>
+            <td>
+              <b>Name</b>
+            </td>
+            <td>
+              <b>Type</b>
+            </td>
+            <td>
+              <b>Description</b>
+            </td>
+          </tr>
+          <tr>
+            <td>Block ID</td>
+            <td>ByteArray[36]</td>
+            <td>The Block ID of the parent.</td>
+          </tr>
+        </table>
+      </details>
+    </td>
+  </tr>
+  <tr>
+    <td>Shallow Like Parents Count</td>
+    <td>uint8</td>
+    <td>The number of blocks following, which are directly referenced to adjust opinion.</td>
+  </tr>
+  <tr>
+    <td valign="top">Shallow Like Parents <code>optAnyOf</code></td>
+    <td colspan="2">
+      <details>
+        <summary>Parent</summary>
+        <blockquote>A reference to a block.</blockquote>
+        <table>
+          <tr>
+            <td>
+              <b>Name</b>
+            </td>
+            <td>
+              <b>Type</b>
+            </td>
+            <td>
+              <b>Description</b>
+            </td>
+          </tr>
+          <tr>
+            <td>Block ID</td>
+            <td>ByteArray[36]</td>
+            <td>The Block ID of the parent.</td>
+          </tr>
+        </table>
+      </details>
+    </td>
+  </tr>
+  <tr>
+    <td>Payload Length</td>
+    <td>uint32</td>
+    <td>The length of the following payload in bytes. A length of 0 means no payload will be attached.</td>
+  </tr>
+  <tr>
+    <td valign="top">Payload <code>optOneOf</code></td>
+    <td colspan="2">
+      <details>
+        <summary>Signed Transaction</summary>
+        <blockquote>A transaction with its unlocks. Defined in <a href='../TIP-0045/tip-0045.md#signed-transaction'>TIP-45 (Signed Transaction)</a>.</blockquote>
+      </details>
+      <details>
+        <summary>Tagged Data</summary>
+        <blockquote>Optional Data with an optional Tag. Defined in <a href='../TIP-0053/tip-0053.md#tagged-data'>TIP-53 (Tagged Data)</a>.</blockquote>
+      </details>
+      <details>
+        <summary>Candidacy Announcement</summary>
+        <blockquote>Signals candidacy for committee selection for the epoch after the one in which it is issued. Defined in <a href='../TIP-0040/tip-0040.md#candidacy-announcement'>TIP-40 (Candidacy Announcement)</a>.</blockquote>
+      </details>
+    </td>
+  </tr>
+  <tr>
+    <td>Max Burned Mana</td>
+    <td>uint64</td>
+    <td>The amount of Mana the Account identified by <code>Issuer ID</code> is at most willing to burn for this block. The actual Mana deducted from this Account's Block Issuance Credit may be lower than the value of this field which is the product of the block's work score and the RMC (Reference Mana Cost) from the block's slot commitment, identified by the <code>Slot Commitment ID</code>. Therefore, for the calculation of this field, the block issuer should also use the RMC value from the same commitment.</td>
```

Comment:
`...is the product of the block's work score and the RMC...`
This is not super clear if we are speaking about the work score of the block or of the basic block body. Can you clarify please?

Reply:
It's the work score of the "outer" block, i.e. the _block_ not the _block body_. Not sure how to clarify better in the TIP, since block is the most accurate term there is for this structure. Do you have a suggestion by chance?

Reply:
It's a bit dumb but we could say something like `block (not block body)`, better be as explicit as we can

Reply:
Clarified this part with a link to the work score calculation which explicitly mentions the signature, so it should be clear.